src/devtools/client/debugger/src/actions/sources/newSources.js

```diff
@@ -86,6 +86,11 @@ function checkPendingBreakpoints(cx, sourceId) {
       return;
     }
 
+    for (const bp of pendingBreakpoints) {
+      const line = bp.location.line;
+      dispatch({ type: "SET_REQUESTED_BREAKPOINT", location: { sourceId, line } });
```

Comment:
why not include the column as well?

Reply:
I basically copied this from the only other place where a `SET_REQUESTED_BREAKPOINT` action is dispatched (in `modify.js`)

Reply:
weird - feels like an omission, but tough to say...

Reply:
It does, but I found this is actually enforced [here](https://github.com/RecordReplay/devtools/blob/b4a2ce306178b3f1eb5b4d420b9dc18bfdf6abcb/src/devtools/client/debugger/src/reducers/breakpoints.js#L77)

Reply:
weird...